hotel_housekeeping/tests/test_hotel_housekeeping.py

```diff
@@ -0,0 +1,80 @@
+# See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import common
+from datetime import datetime
+import time
+
+
+class TestHousekeeping(common.TransactionCase):
+
+    def setUp(self):
+        super(TestHousekeeping, self).setUp()
+        self.housekeeping_obj = self.env['hotel.housekeeping']
```

Comment:
would be nice to see `tracking_disable` in a global context, you can achieve it by using
```python
self.env = self.env(context=dict(
    self.env.context,
    'tracking_disable': True,
))
```